src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder();
+        try {
+            LocalDate dateStart = LocalDate.parse(dateFrom, formatter);
+            LocalDate dateFinish = LocalDate.parse(dateTo, formatter);
+            int salary = 0;
+            builder.append("Report for period " + dateFrom + " - " + dateTo);
```

Comment:
Here you still have `String` concatenation, which can be avoided by chaining `append()` methods.